### PR TITLE
Implemented image gallery and header

### PR DESCRIPTION
**Pull request by handelc22**

### CODE REVIEW

File: client/src/components/ProductDetails/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Carousel extends React.Component {
```
Comment (abachant) [inline client/src/components/ProductDetails/Carousel.jsx:6]: I think this could benefit from more comments

**Review by abachant — COMMENTED** [review]
Really good stuff!

File: client/src/components/ProductDetails/Carousel.jsx (hunk 1 above)
Comment (handelc22) [reply client/src/components/ProductDetails/Carousel.jsx:6]: Added more comments throughout the file! Thank you

File: client/src/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  cartOnClick() {
+    const { cartDisplay } = this.state;
+    if (cartDisplay === 'none') {
+      this.setState({ cartDisplay: 'flex' });
+    } else {
+      this.setState({ cartDisplay: 'none' });
+    }
+  }
```
[the remark this answers is not in the document]
Comment (handelc22) [reply client/src/Header.jsx:21]: Good catch! Thank you

**Pull request merged**